test/unit/core/OperationRepo.ts

```diff
@@ -0,0 +1,167 @@
+    t.truthy(executor);
+  });
+});
+
+test("Model repo delta broadcast is received and processed by operation repo", async t => {
+  const clock = sinon.useFakeTimers();
+  const core = new CoreModule();
+  await core.init();
+  const { modelRepo, operationRepo } = core;
+  const executor = operationRepo?.executorStore[ModelName.EmailSubscriptions];
+
+  const broadcastSpy = sinonSandbox.spy(ModelRepo.prototype, "broadcast");
+  const processDeltaSpy = sinonSandbox.spy(OperationRepo.prototype as any, "processDelta");
+
+  modelRepo?.broadcast({
+    changeType: CoreChangeType.Add,
+    model: new OSModel(ModelName.EmailSubscriptions),
+  });
+
+  clock.tick(1001);
+
+  t.true(broadcastSpy.calledOnce);
+  t.true(processDeltaSpy.calledOnce);
+  t.is(executor?.operationQueue.length, 1);
+  t.is(executor?.deltaQueue.length, 0);
+});
+
+test("Add Subscriptions: Multiple delta broadcasts -> two operations of change type: add", async t => {
+  const clock = sinon.useFakeTimers();
+  const core = new CoreModule();
+  await core.init();
+  const { modelRepo, operationRepo } = core;
+  const executor = operationRepo?.executorStore[ModelName.EmailSubscriptions];
+
+  const broadcastSpy = sinonSandbox.spy(ModelRepo.prototype, "broadcast");
+  const processDeltaSpy = sinonSandbox.spy(OperationRepo.prototype as any, "processDelta");
+
+  modelRepo?.broadcast({
+    changeType: CoreChangeType.Add,
+    model: new OSModel(ModelName.EmailSubscriptions),
+  });
+  modelRepo?.broadcast({
+    changeType: CoreChangeType.Add,
+    model: new OSModel(ModelName.EmailSubscriptions),
+  });
+
+  clock.tick(1001);
+
+  t.true(broadcastSpy.calledTwice);
+  t.true(processDeltaSpy.calledTwice);
+  t.is(executor?.operationQueue.length, 2);
+  t.is(executor?.deltaQueue.length, 0);
+});
+
+test("Update Identity: Multiple delta broadcasts -> one operation of change type: update", async t => {
+  const clock = sinon.useFakeTimers();
+  const core = new CoreModule();
+  await core.init();
+  const { modelRepo, operationRepo } = core;
+  const executor = operationRepo?.executorStore[ModelName.Identity];
+
+  const broadcastSpy = sinonSandbox.spy(ModelRepo.prototype, "broadcast");
+  const processDeltaSpy = sinonSandbox.spy(OperationRepo.prototype as any, "processDelta");
+
+  const delta1: CoreDelta<IdentityModel> = {
+    changeType: CoreChangeType.Update,
+    model: new OSModel(ModelName.Identity),
+    property: "myAlias",
+    oldValue: "",
+    newValue: "myNewAlias",
+  };
+
+  const delta2: CoreDelta<IdentityModel> = {
+    changeType: CoreChangeType.Update,
+    model: new OSModel(ModelName.Identity),
+    property: "myAlias2",
+    oldValue: "",
+    newValue: "myNewAlias2",
+  };
+
+  modelRepo?.broadcast(delta1);
+  modelRepo?.broadcast(delta2);
+
+  clock.tick(1001);
+
+  t.true(broadcastSpy.calledTwice);
+  t.true(processDeltaSpy.calledTwice);
+  t.is(executor?.operationQueue.length, 1);
+  t.is(executor?.deltaQueue.length, 0);
+});
```

Comment:
see if it makes sense to also inspect the changes on the operation itself (alias values)